target/bin/debug-getmail

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+
+# shellcheck source=../scripts/helpers/log.sh
+source /usr/local/bin/helpers/log.sh
+# shellcheck source=../scripts/startup/setup-stack.sh
+source /usr/local/bin/setup-stack.sh
```

Comment:
Just realized: the current error is related here. You'll probably not source this stack, but the corresponding setup script (<[target/scripts/startup/setup.d/getmail.sh](https://github.com/docker-mailserver/docker-mailserver/pull/2803/files#diff-9103a52f7cd4812e074e2240f8628053ba0d7ea91402c9f48b13c06f85fea06f)>) I guess.

Reply:
fixed that ...l my bad ... I'd done this on my local branch but overwrote it with the version you checked in :)